docs/installing-northstar/northstar-installers/viper-guide.md

```diff
@@ -0,0 +1,65 @@
+# Viper Guide
+
+Viper is a mod manager/installer for Northstar, a community made mod for Titanfall 2, that automatically updates mods and Northstar versions.
+
+Similar to FlightCore, Viper works on both Windows and Linux and have similar use cases, mostly differing in downloading Viper.
+
+## Windows Installation
+
+![The Windows download button on Viper's GitHub page](../../images/viper-windows-download.png)
+
+Install Viper from the [Northstar website](https://northstar.tf), the newest `.exe` file from [Viper's GitHub releases page](https://github.com/0neGal/viper/releases/), or the Windows download button on [Viper's main GitHub page](https://github.com/0neGal/viper).
```

Comment:
Might actually omit the GitHub release page part as it offers both the installer `Viper-Setup.exe` and the portable version `Viper.exe` and the portable version doesn't auto-update AFAIK, which means there will eventually be issues with people using outdated version when running that one ^^